src/proxyProvider.ts

```diff
@@ -26,8 +26,14 @@ export class ProxyProvider implements IProvider {
      */
     constructor(url: string, config?: AxiosRequestConfig) {
         this.url = url;
+        // See: https://github.com/axios/axios/issues/983
         this.config = config || {
```

Comment:
I guess we should use `Object.assign` or something to merge the provided config and the defaults so that one default won't get lost when a `config` is passed as a parameter.

Reply:
Ping @pctj101 - Do you think you can implement the requested changes? Or should we do it - there's no problem either way.
Then we can merge this PR :)

Reply:
Oh hi!  I was caught up in a few things but I can get to it later today :)  Thanks for saying hi!

Reply:
pushed up a change, but definitely take a look at the data type comment below in case you have any insight.

Reply:
Thanks a lot!